src/foam/nanos/auth/UserAndGroupAuthService.js

```diff
@@ -257,8 +247,8 @@ foam.CLASS({
     {
       name: 'validatePassword',
       javaCode: `
-        if ( SafetyUtil.isEmpty(potentialPassword) || ! (Pattern.compile(PASSWORD_VALIDATE_REGEX)).matcher(potentialPassword).matches() ) {
-          throw new RuntimeException(PASSWORD_VALIDATION_ERROR_MESSAGE);
+        if ( potentialPassword.length() < 6 ) {
+          throw new RuntimeException("PASSWORD_VALIDATION_ERROR");
```

Comment:
Why remove the isEmpty() check?  That protects against password being null.  Additionally why switch to a less informative error message?

Reply:
Point taken.

PASSWORD_VALIDATION_ERROR_MESSAGE put back and isEmpty check added to if statement

The reason I changed was on the thought of simplifying. There is work that needs to be done with passwords outlined in CPF-2259